Madek/madek
# release: v3.22.0

todo
- [x] check translation fallbacks
- [x] batch translation enhancements

dev stuff

- [x] 75cb881097814522b5a89a7800d64710175de0f3 deploy: change auto-deploy branch for staging to zhdk/staging => CI only
- [x] f8485017a8908fc6dfacd4b246494c68172e1d76 migration: helpers for assistant
- [x] 69386707a01c5c889ea8f43641698cf66e965df2 ci: git lint: allow merges when from stable release
- [x] eb02c02a2daddbe69d0d83643830769e9004cd4a ci: fix meta check
- [x] 8e454e2e5a333fc4cf84db5386f465252b751acd Merge branch 'release'

to test

- [x] a5cd1a28f96cadbb9273d2bb49d8b6791d9a48ce release: v3.22.0-RC.1 ➡️ **braucht noch einen Namen**
- [x] ba515b358048593b8a67826c5f2922acd2363bff webapp: translations: update from corrections
- [x] 2be19add0ea6ee7689dca7f4ca1618ef93c2ee95 datalayer: translate madek core vocab
    - see here: <https://test-blank.madek.zhdk.ch/vocabulary/madek_core?lang=en>
- [x] f3b40d748051a8e86fd47e5cef4181868ea4f093 api: remove 'creator' from Entry, Set, Keyword 
  - dont spend too much time, its a simple API change. maybe check exporter output (keywords)
- [x] 276045210c4a86e24e2ac5c5f64027736aca3863 webapp: fix: disable set cover zooming
  - already approved
- [x] 770396384e7a24389e6517fa3e90e3ec650d13b0 webapp: fix: view: PersonShow: limit line lengths
- [x] 97862e43989cc986a84329d46d51708e10f88706 webapp: fix: use configured external URL for sharing links
  - click on share, shows same URL as in browser <https://staging.madek.hfg-karlsruhe.de/entries/ca9434f1-a914-4892-85eb-3196300cde41>
- [x] 673cff367604c57b73bba622dea52c1d00f737a6 admin: feat: filter deactivated users, identify them with proper label ➡️ as show on last demo
- [x] 836b669eb98024f79517884c114f6b10a31961ee webapp: fix: show Users as deactivated in Permission forms ➡️ as show on last demo
- [x] 34d8e3c51d0fc59bf88380221fa1442549137ea3 sql-assistant: find ambigously named people ➡️ as show on last demo
- [x] 7da5225912983df768716144e9e0e67e44ca3a92 feat: show email in User-Autocomplete ➡️ as show on last demo
- [x] f8face89cdef0d122ed4be5dd27f213aa69b6bb2 feat: People w/ description and URI ➡️ as show on last demo
- [x] 2b5125f  admin: feat: batch translation: fold section, sort by position ➡️ enhancement ideas from @ltsotrug
